__tests__/app.test.js

```diff
@@ -58,6 +58,27 @@ describe("ERROR Handling", () => {
         expect(body.msg).toBe("Article not found!");
       });
   });
+  test("status:400 for a request where the object is empty", () => {
+    return request(app)
+      .patch("/api/articles/2")
+      .send({})
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Invalid request!");
+      });
+  });
+  test.only("status: 400 for invalid inc_votes property type", () => {
+    const articleUpdate = {
+      inc_votes: "Mr. Potatoe Head",
+    };
+    return request(app)
+      .patch("/api/articles/3")
+      .send(articleUpdate)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Invalid request!");
+      });
+  });
```

Comment:
nicely done however we still need a test for a non-existing article id (9999)

Reply:
that test is in the Error handling section I believe?

Reply:
test("status: 404 for a valid, but non existing article_id", () => {
    return request(app)
      .get("/api/articles/9000")
      .expect(404)
      .then(({ body }) => {
        expect(body.msg).toBe("Article not found!");
      });
  });
});

this is copied from the Error handling section in my tests, is this not the test you mean?